src/default_config_params/mod.rs

```diff
@@ -0,0 +1,66 @@
+use std::path::PathBuf;
+pub fn get_default_wallet_path() -> PathBuf {
+    let home_var = std::env::var("HOME");
+    match home_var {
+        Ok(home_path) => {
+            let teleport_config_dir = PathBuf::from(home_path).join(".teleport");
+            if teleport_config_dir.exists() {
+                let wallet_path = teleport_config_dir.join("wallet.teleport");
+
+                wallet_path
+            } else {
+                std::fs::create_dir_all(teleport_config_dir.clone()).unwrap();
+                let wallet_path = teleport_config_dir.join("wallet.teleport");
+
+                wallet_path
+            }
+        }
+        Err(e) => {
+            println!("HOME env variable not set on this Terminal instance. {e}. Creating wallet in default project path");
+
+            let wallet_path = PathBuf::from("wallet.teleport");
+            wallet_path
+            // match wallet_path {
+            //     Ok(wallet_path) => wallet_path,
+            //     Err(e) => {
+            //         println!("Unable to create a wallet file in the default location: {e}");
+            //         panic!()
+            //     }
+            // }
+        }
+    }
+}
+
+#[cfg(test)]
+mod test_default_config_params {
+    use std::path::PathBuf;
+
+    use crate::default_config_params::get_default_wallet_path;
+    //TODO investigate on environment variables for test cases
+
+    #[test]
+    // Test on failure to get HOME variable: system should create the wallet in the current dir
+    // Test on incorrect HOME variable: system should create the wallet in the current dir
```

Comment:
The test cases should cover. 

- load/write the `WalletStore` from an user defined datadir.
- load/write the `WalletStore` from the default datadir.

Tests should be included in the `wallet/store.rs` module.

Reply:
At what point does the user get to define their preferred datadir? Would this be from the command-line arguments?